syft/execution/placeholder.py

```diff
@@ -34,7 +34,7 @@ def instantiate(self, tensor):
         """
         if isinstance(tensor, PlaceHolder):
             self.child = tensor.child
-        elif tensor.is_wrapper:
+        elif hasattr(tensor, "is_wrapper") and tensor.is_wrapper:
```

Comment:
This is needed because after we built a plan the ```tensor``` (received as a parameter) is a ```MPCTensor``` (does not have the attribute ```is_wrapper```.
Another solution can be seen in a previous commit [here](https://github.com/OpenMined/PySyft/pull/3254/commits/d884650282b6cd45c8c80562cb431d9a81310988#diff-3fd9f3371ec1373832b99032d1212bad) (I kinda like more this approach)

Reply:
Either way is probably fine. Using `hasattr` protects us against future tensor types that don't have the attribute though.